src/components/main/StoreDetailSection/Content/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { useSearchParams } from 'next/navigation';
+
+import Report from '../Report';
+import Reviews from '../Reviews';
+
+import Header from '@components/common/Header';
+import ImageContainer from '@components/common/ImageContainer';
+import { useGetReport } from '@hooks/api/useGetReport';
+import useObserver from '@hooks/useObserver';
+
+export default function StoreDetailContent() {
+  const searchParams = useSearchParams();
+
+  const storeId = searchParams.get('storeId');
+
+  const { data } = useGetReport(storeId ?? '');
+
+  const [isScrollDown, setIsScrollDown] = useState(false);
+
+  const onIntersect: IntersectionObserverCallback = ([entry]) => {
+    setIsScrollDown(!entry.isIntersecting);
+  };
+
+  const { setTarget } = useObserver({
+    onIntersect,
+  });
+
+  return (
+    <div>
+      <div ref={setTarget}>
+        <Header
+          className={`fixed bg-white ${
+            data?.data.storeMainImageUrl && '[&>*>*]:fill-white'
+          }`}
+        >
```

Comment:
cn을 사용하는게 쫌 더 안전할 듯 합니다!
```suggestion
        <Header
          className={cn("fixed bg-white", { '[&>*>*]:fill-white' :  data?.data.storeMainImageUrl })}
        >
```